subsystem/cluster_test.go

```diff
@@ -217,7 +219,7 @@ func waitForClusterInstallationToStart(clusterID strfmt.UUID) {
 	waitForClusterState(context.Background(), clusterID, models.ClusterStatusPreparingForInstallation,
 		10*time.Second, IgnoreStateInfo)
 	waitForClusterState(context.Background(), clusterID, models.ClusterStatusInstalling,
-		180*time.Second, "Installation in progress")
+		300*time.Second, "Installation in progress")
```

Comment:
Two tests were timing out here, and I found that they just needed a bit more time to pass.